src/components/Modal/Modal.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import InfoIcon from '@mui/icons-material/Info';
+import './Modal.css';
+import wordle01 from './wordlesolver01.png';
+import wordle02 from './wordlesolver02.png';
+import wordle03 from './wordlesolver03.png';
+
+export default function Modal({ modal, setModal }) {
+
+    const toggleModal = () => {
+        setModal(!modal);
+    };
+
+    if (modal) {
+        document.body.classList.add('active-modal');
```

Comment:
this is a very bad practice in React (directly manipulating dom elements)

A more appropriate way to do this is 
<div className={active ? "modal-active" : "modal"} />

An even better way to do this is don't render the modal at all unless it's open or you want to persist data in it for some reason.
```ts
function Modal({ setModal }) {
   ...
}
```

```ts
const ComponentContainingAModal = ({}) => {
   const [modal, setModal] = useState();

   if (!modal) {
       return null;
   }

   return <Modal .../>
   }
```